internal/support/notifications/application/notification.go

```diff
@@ -146,7 +146,11 @@ func DeleteNotificationById(id string, dic *di.Container) errors.EdgeX {
 		return errors.NewCommonEdgeX(errors.KindContractInvalid, "id is empty", nil)
 	}
 	dbClient := container.DBClientFrom(dic.Get)
-	err := dbClient.DeleteNotificationById(id)
+	_, err := dbClient.NotificationById(id)
+	if err != nil {
+		return errors.NewCommonEdgeXWrapper(err)
+	}
```

Comment:
move this check to the application level

Reply:
This is already the application level.